test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt

```diff
@@ -20,962 +20,972 @@
+com/sun/crypto/provider/Mac/MacKAT.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/Mac/MacSameTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/Mac/NullByteBufferTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/NSASuiteB/TestAESOids.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/NSASuiteB/TestAESWrapOids.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/NSASuiteB/TestHmacSHAOids.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/TLS/TestKeyMaterial.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/TLS/TestLeadingZeroes.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/TLS/TestMasterSecret.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/TLS/TestPRF.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/TLS/TestPRF12.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/TLS/TestPremaster.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/org/apache/xml/internal/security/ShortECDSA.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/org/apache/xml/internal/security/SignatureKeyInfo.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/org/apache/xml/internal/security/TruncateHMAC.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+java/foreign/TestFallbackLookup.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+java/foreign/TestMismatch.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
```

Comment:
These don't seem related to FIPS140-3 and they're not listed in other exclude lists: why are they here?
Same question for:
* java/lang/Thread/virtual/stress/GetStackTraceALotWhenPinned.java
* serviceability/jvmti/vthread/GetThreadState/GetThreadStateTest.java
* serviceability/jvmti/vthread/GetThreadState/GetThreadStateTest.java
* serviceability/jvmti/vthread/SuspendResume2/SuspendResume2.java

Reply:
1. Failures are not identical among various JDK versions. 2. Those failures shows up while running in FIPS 140-3 mode. We had a discussion with the test team, test samples in some folders do not need to be run, so failed samples in these folders can be eliminated.

Reply:
I was referring to other exclude lists for this repository (same version).

Reply:
Im not quite sure what do you mean by other exclude lists for this repository. There is only one exclude list file for fips 140-3 right now.

Reply:
Removed those unrelated failures.